SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialClassInfoManager.cpp

```diff
@@ -331,7 +331,11 @@ void USpatialClassInfoManager::FinishConstructingSubobjectClassInfo(const FStrin
 		SpecificDynamicSubobjectInfo->bDynamicSubobject = true;
 
 		int32 Offset = DynamicSubobjectData.SchemaComponents[SCHEMA_Data];
-		check(Offset != SpatialConstants::INVALID_COMPONENT_ID);
+		if (!ensureAlwaysMsgf(Offset != SpatialConstants::INVALID_COMPONENT_ID,
```

Comment:
This could indicate an out-of-date schema database, which might extend beyond the scope of this method.

Reply:
We should hit a schema mismatch on connection if that's the case?

Reply:
That's only a mismatch between server and client. The out-of-date I'm talking about is between the Unreal asset/classes and the computed schema.